forem/forem
# Change theme after update without manual refreshing

## What type of PR is this? (check all applicable)


## Description

## Related Tickets & Documents
https://github.com/forem/forem/issues/3613

## QA Instructions, Screenshots, Recordings
- Go to User's Settings page. 
- Navigate to UX tab, select some theme.
- Theme should get updated website wide.

## Added tests?


## Added to documentation?
